packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -287,6 +287,15 @@ contract EpochRewards is
     return true;
   }
 
+  function setTargetVotingYield(uint256 targetVotingYield) public onlyOwner {
+    FixidityLib.Fraction memory target = FixidityLib.wrap(targetVotingYield);
+    require(
+      target.lte(targetVotingYieldParams.max),
+      "Target voting yield must be less than or equal to max"
+    );
+    targetVotingYieldParams.target = target;
+  }
```

Comment:
Should we `return true;` here? Seems to be a convention we adopted.

Reply:
yup -- i updated